lib/theme/widgets/hs_text_field.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+
+InputDecoration inputTextTheme = InputDecoration(
+    contentPadding: const EdgeInsets.fromLTRB(16, 13, 12, 13),
+    border: OutlineInputBorder(
+        borderRadius: BorderRadius.circular(8),
+        borderSide: const BorderSide(
+          color: HSColor.black,
+        )),
+    hintText: HatSpaceStrings.current.pleaseEnterYourPlaceholder,
+    hintStyle:
+        textTheme.bodyMedium?.copyWith(height: 1.0, color: HSColor.neutral5));
+
+class HatSpaceLabel extends StatelessWidget {
+  final String label;
+  final bool isRequired;
+  final String? optional;
+  const HatSpaceLabel({
+    super.key,
+    required this.label,
+    required this.isRequired,
+    this.optional,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    // TODO: implement build
+    return RichText(
+        text: TextSpan(
+            style: Theme.of(context).textTheme.bodyMedium,
+            text: label,
+            children: [
+          const TextSpan(text: " "),
+          TextSpan(
+              text: isRequired ? " *" : optional ?? "",
+              style: textTheme.bodyMedium
+                  ?.copyWith(color: isRequired ? HSColor.requiredField : null))
```

Comment:
this means the label will always have a white space at the end, is it needed to have extra `const TextSpan(text: " ")`?